src/changelog/unreleased.md

```diff
@@ -41,3 +41,4 @@
 - Add `Window::default_attributes` to get default `WindowAttributes`.
 - `log` has been replaced with `tracing`. The old behavior can be emulated by setting the `log` feature on the `tracing` crate.
 - On Windows, confine cursor to center of window when grabbed and hidden.
+- On iOS: For IOS/UIKit Support UIGestureRecognizerDelegate for simultaneous gesture input
```

Comment:
Could you clean up the PR title, PR description and this changelog entry, being a bit more detailed on what this adds? Perhaps in more generic terms ("Add `PanGesture` window event and support it on iOS").

Reply:
Nit: Perhaps I'd rather have this merged with the "on iOS, add detection support for `PinchGesture`, `DoubleTapGesture` and `RotationGesture`" entry further above in this file; for users, it's not really relevant what modifications were made, since the release has yet to be cut.